src/sentry/templates/sentry/integrations/jira-server-config.html

```diff
@@ -0,0 +1,66 @@
+{% extends "sentry/bases/modal.html" %}
+{% load crispy_forms_tags %}
+{% load sentry_assets %}
+{% load i18n %}
+
+{% block css %}
+  <link href="{% asset_url "sentry" "dist/sentry.css" %}" rel="stylesheet"/>
+  <style>
+    ol li {
+      padding:10px 0px;
+    }
+    .code-list {
+      padding: 10px;
+      background-color: #f7f8f9;
+      margin-top: 10px;
+      border-radius: 3px;
+    }
+    .code-list li {
+      font-size: 90%;
+    }
+  </style>
+{% endblock %}
+
+{% block wrapperclass %} narrow auth {% endblock %}
+{% block modal_header_signout %} {% endblock %}
+
+{% block title %} {% trans "Jira-Server Setup" %} | {{ block.super }} {% endblock %}
+
+{% block main %}
+{% if form %}
```

Comment:
@markstory I think if these are going to be a larger trend, then I feel it wouldn't hurt to do some more .... template inheritance and such here. It's not high priority, but I'm thinking about it....

Reply:
I agree. We can revisit gitlab and this next week.